docker.local/config/conductor.config.yaml

```diff
@@ -578,6 +581,16 @@ commands:
       remotepath:
         required: false
         default: /remote/upload.bin
+  add_hardfork:
+    work_dir: "../"
+    exec: "sh -x 0chain/docker.local/bin/conductor/add_hardfork.sh $remotepath"
```

Comment:
Should pass rounds and names here: `sh -x 0chain/docker.local/bin/conductor/add_hardfork.sh $names $rounds`. They will be autofilled based on the definition of the parameters.